data_model/src/account.rs

```diff
@@ -237,20 +235,18 @@ declare_item! {
     #[display(fmt = "({id})")] // TODO: Add more?
     pub struct Account {
         /// An Identification of the [`Account`].
-        id: Id,
+        pub id: Id,
         /// Assets in this [`Account`].
-        assets: AssetsMap,
+        pub assets: AssetsMap,
```

Comment:
So AssetsMap is itself opaque, but we can assign the assets field and don't need to go through two function calls to modify the map. Only one of those two calls would go through a VTable, so I wonder how much do we actually save.